client/src/components/DropdownPace/DropdownPace.js

```diff
@@ -0,0 +1,53 @@
+import styled from "styled-components/macro";
+import PropTypes from "prop-types";
+const Select = styled.select`
+  height: 40px;
+  width: 100%;
+  margin-top: 15px;
+  border: solid 2px var(--secondary-color);
+  background: none;
+  color: var(--secondary-color);
+  font-family: "Heiti SC", adobe-heiti-std, sans-serif;
+  font-size: 1rem;
+  font-weight: 700;
+  display: flex;
+  text-align: center;
+`;
+
+const DropdownPace = ({ pace, setPace }) => {
+  const handlePacePick = (event) => {
+    setPace(event.target.value);
+  };
+
+  return (
+    <Select onChange={handlePacePick} value={pace}>
+      <option pace="2.5 min/km"> 2.0 min/km</option>
+      <option pace="2.5 min/km"> 2.5 min/km</option>
```

Comment:
`option` doesn't have `pace`.

```suggestion
      <option value="2.5 min/km"> 2.5 min/km</option>
```